main.js

```diff
@@ -41,9 +38,14 @@ app.on('window-all-closed', function () {
   if (process.platform !== 'darwin') app.quit()
 })
 
-function finishTest (success) {
-  process.exit(success ? 0 : 1)
-}
-function failTestIfBadProcess (details) {
-  if (details.reason !== 'clean-exit') finishTest(false)
+// Test helpers
+const test = {
+  done: (success) => process.exit(success ? 0 : 1),
+  init: () => {
+    crashReporter.start({ uploadToServer: false, submitURL: '' })
```

Comment:
Ideally we would call `crashReporter.start` even before `app.whenReady()`, to catch any super early crashes.

Reply:
ok. the `test` object isn't defined unless we move it to the top of the file, so instead of having a `test.init()` function I put an IIFE at the end of the file.

This left `test.done` as the only property in main's `test`, so I removed the object altogether.

Fixed 9b3322369caab7af28e58c84726c2fed8c63eb42